x-pack/plugins/fleet/server/routes/setup/index.ts

```diff
@@ -26,14 +25,15 @@ export const registerFleetSetupRoute = (router: IRouter) => {
   );
 };
 
+// That route is used by agent to setup Fleet
 export const registerCreateFleetSetupRoute = (router: IRouter) => {
   router.post(
     {
       path: AGENTS_SETUP_API_ROUTES.CREATE_PATTERN,
-      validate: PostFleetSetupRequestSchema,
+      validate: false,
       options: { tags: [`access:${PLUGIN_ID}-all`] },
     },
-    fleetAgentSetupHandler
+    fleetSetupHandler
```

Comment:
we are already registering this handler on another route above, `registerFleetSetupRoute`. should we remove one of these?

Reply:
the `/agents/setup` is currently used by the agent. I think it's okay to have both and it will keep us in a place where we can have specific setup for agents in the future. Also removing it will be a breaking change for the existing agent

Reply:
> removing it will be a breaking change for the existing agent

what do you mean by this? I thought we were ok with breaking changes for existing agents since we are not offering a migration path. aren't we unenrolling existing agents on upgrade to 7.13?

Reply:
It will break the agent on master that is using that setup call.